src/website/views/components/date-picker/_examples.pug

```diff
@@ -182,6 +182,45 @@ p.-text
       <chi-date-picker id="date-excluded-days-dates" excluded-weekdays="0, 6" excluded-dates="02/23/2024, 05/24/2024, 07/03/2024, 09/12/2024, 12/12/2024"></chi-date-picker>
   #html-excluded-days-dates.chi-tabs-panel
 
+h3 Message
+p.-text
+  | Use the <code>helper-message</code> attribute to provide users with additional information, validation feedback, and other helpful information.
+.example.-mb--3
+  .-p--3
+    div(style="max-width: 14rem;")
+      chi-label(for='date-message')
+        | Date
+      chi-date-picker(id="date-message" helper-message="Optional helper message")
+
+  .example-tabs.-pl--2
+    ul.chi-tabs#example-with-message
+      li.-active
+        a(href=`#webcomponent-with-message`) Web Component
+      li
+        a(href=`#html-with-message`) HTML Blueprint
+  #webcomponent-with-message.chi-tabs-panel.-active
+    :code(lang="html",style="height:20rem")
```

Comment:
I know this is in several examples, but does this affect anything? If not could this style be removed?

```suggestion
    :code(lang="html")
```